cfme/tests/integration/test_openldap_auth.py

```diff
@@ -30,3 +38,6 @@ def test_openldap_auth(request, setup_first_provider, configure_openldap_auth_mo
     with user:
         login.login(user)
         assert login.current_full_name() == data["fullname"]
+        login.logout()
+    login.login_admin()
+    assert user.is_user_present() is True
```

Comment:
...so this would turn into `assert user.exists is True` (exists should be a property)